tips/TIP-0025/core-rest-api.yaml

```diff
@@ -0,0 +1,3330 @@
+            tokenSupply:
+              type: string
+              description: Current supply of base token. Plain string encoded number.
+            version:
+              type: integer
+              description: Protocol version used by the network.
+            minPowScore:
+              description: The Proof-of-Work difficulty for a block to be sent over the network to mitigate spam.
+              type: number
+              format: float
+            rentStructure:
+              description: The rent structure according to TIP-19.
+              properties:
+                vByteCost:
+                  description: Defines the rent of a single virtual byte denoted in IOTA tokens.
+                  type: integer
+                vByteFactorData:
+                  description: Defines the factor to be used for data only fields.
+                  type: integer
+                vByteFactoKey:
+                  description: Defines the factor to be used for key/lookup generating fields.
+                  type: integer
+              required:
+                - vByteCost
+                - vByteFactorData
+                - vByteFactoKey
+          required:
+            - networkName
+            - bech32Hrp
+            - tokenSupply
+            - protocolVersion
+            - minPowScore
+            - rentStructure
+        pendingProtocolParameters:
+          description: Pending protocol parameters.
+          type: array
+          items:
+            $ref: '#/components/schemas/ProtocolParamsMilestoneOpt'
+        baseToken:
+          description: Gives info about the base token the network uses.
+          properties:
+            name:
+              type: string
+              description: The name of the base token of the network.
+            tickerSymbol:
+              type: string
+              description: Ticker symbol of the token to be displayed on trading platforms.
+            unit:
+              type: string
+              description: The primary unit of the token.
+            decimals:
+              type: integer
+              description: Number of decimals the primary unit is divisible up to.
+            subunit:
+              type: string
+              description: The name of the smallest possible denomination of the primary unit. subunit * 10^decimals = unit
+            useMetricPrefix:
+              type: boolean
+              description: Whether to use metric prefixes for displaying unit.
+          required:
+            - name
+            - tickerSymbol
+            - unit
+            - decimals
+            - useMetricPrefix
+        features:
+          description: The features that are supported by the node. For example, a node could support the Proof-of-Work (PoW) feature, which would allow the PoW to be performed by the node itself.
```

Comment:
```suggestion
          description: The features that are supported by the node. For example, a node could support the Proof-of-Work (Pow) feature, which would allow the PoW to be performed by the node itself.
```